modules/rosa-upgrading-manual-ocm.adoc

```diff
@@ -9,13 +9,13 @@ endif::[]
 
 :_content-type: PROCEDURE
 [id="rosa-upgrade-ocm_{context}"]
-= Upgrading using the console
+= Scheduling individual upgrades through the {cluster-manager} console
 
-You can upgrade a {product-title} cluster
+You can schedule to upgrade a {product-title} cluster
```

Comment:
_You can schedule...just once_ sounds a bit scary. It is that you only need to schedule the upgrade one time or you have only one chance to schedule the upgrade?

Reply:
The former is correct. Changed 'just once' to 'one time'